lang/python/2to3/Makefile

```
@@ -0,0 +1,54 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=2to3
+PKG_VERSION:=1.0
+PKG_RELEASE:=1
+
+PKG_MAINTAINER:=Gerald Kerma <gandalf@gk2.net>
+
+include $(INCLUDE_DIR)/host-build.mk
+include $(INCLUDE_DIR)/package.mk
+include ../python3-package.mk
+
+define Package/2to3
+  SUBMENU:=Python
+  SECTION:=lang
+  CATEGORY:=Languages
+  TITLE:=2to3 binary using python3
+  DEPENDS:= \
+	  +python3-lib2to3 \
+      +python3-light
+endef
+
+define Package/2to3/description
+ 2to3 is a Python program that reads Python 2.x source code and applies a
+ series of fixers to transform it into valid Python 3.x code. The standard
+ library contains a rich set of fixers that will handle almost all code. 2to3
+ supporting library lib2to3 is, however, a flexible and generic library, so it
+ is possible to write your own fixers for 2to3. lib2to3 could also be adapted
+ to custom applications in which Python code needs to be edited automatically.
+endef
+
+define Package/2to3/Install
+	$(INSTALL_DIR) $(1)/usr/bin
+	$(INSTALL_BIN) ./files/2to3.py $(1)/usr/bin/2to3.py
+endef
+
+define Host/2to3/Install
+	$(INSTALL_DIR) $(STAGING_DIR_HOST)/usr/bin
+    $(INSTALL_BIN) ./files/2to3.py $(STAGING_DIR_HOST)/usr/bin/2to3.py
+endef
+
+define Host/2to3/Clean
+	rm -f $(STAGING_DIR_HOST)/usr/bin/2to3.py
```

```suggestion
	rm -f $(STAGING_DIR_HOSTPKG)/bin/2to3
```

`$(1)` in `Host/Install` is `$(STAGING_DIR_HOSTPKG)` (for packages).